controllers/club/settings/invite_members/impl/emailNewMembers.js

```diff
@@ -43,13 +47,27 @@ module.exports = async (req, res) => {
                         const settings_message = {User: user.firstName + " " + user.lastName, Type: `Invited user via email to join club`, Date: date, Time: time}
                         club.settings_history.unshift(settings_message)
                         club.save()
+                        
+                        const oauth2Client = new OAuth2(
+                            process.env.CLIENTID, // ClientID
+                            process.env.CLIENTSECRET, // Client Secret
+                            "https://developers.google.com/oauthplayground" // Redirect URL
+                       );
+
+                        oauth2Client.setCredentials({
+                        refresh_token: process.env.REFRESHTOKEN
+                        });
+                        const accessToken = oauth2Client.getAccessToken()
+                        
                         const transporter = await nodemailer.createTransport({
```

Comment:
same here, this could probably go into its own function

Reply:
again why/where do you mean to separate the code into separate functions? createTransport and new OAuth2 are built in functions already